src/OVAL/public/oval_definitions.h

```diff
@@ -2450,6 +2450,11 @@ void oval_state_content_set_varcheck(struct oval_state_content *, oval_check_t);
  * @memberof oval_state_content
  */
 void oval_state_content_set_entcheck(struct oval_state_content *, oval_check_t);
+/**
```

Comment:
Since it's public API it would be good to provide more documentation for doxygen like comment the parameter and return value. Even if it's obvious to us it won't be obvious to someone who will want to try build something on top of it using only doxygen doc.